src/modules/admin/components/formRules/FormRule.tsx

```diff
@@ -26,11 +18,25 @@ const ActiveChip = ({ active }: { active: boolean }) => (
   />
 );
 
-const FormRule: React.FC<Props> = ({
-  rule,
-  setSelectedRule,
-  openFormDialog,
-}) => {
+const nonClientFormRoles = [
+  FormRole.CeParticipation,
+  FormRole.Funder,
+  FormRole.HmisParticipation,
+  FormRole.Inventory,
+  FormRole.Organization,
+  FormRole.Project,
+  FormRole.ProjectCoc,
+  FormRole.ReferralRequest,
+];
+
+const nonProjectFormRoles = [FormRole.Organization];
+
+interface Props {
+  rule: FormRuleFieldsFragment;
+  formRole: FormRole;
+}
+
+const FormRule: React.FC<Props> = ({ rule, formRole }) => {
```

Comment:
> Should this reuse the existing ProjectApplicabilitySummary chip, which I just discovered

That component was just a quick implementation for a way to display project applicability rules. It would be a plus if we could use the same underlying component to render rules in `ProjectConfigTable`. The project config settings are more limited, which is why that component [accepted a union type and used type guards to access the non-overlapping fields](https://github.com/greenriver/hmis-frontend/blob/90336b447c650dad7cc758e0e5ce49fca0c1d796/src/modules/admin/components/formRules/ProjectApplicabilitySummary.tsx#L23-L25). The same approach could be taken with this FormRule component (Eg have the incoming rule prop be `FormRuleFieldsFragment | ProjectConfigFieldsFragment`, and `formRole` be optional). 

But that's purely a nice-to-have code cleanup. We can always do it later.